cluster/gce/gci/configure-helper.sh

```diff
@@ -2112,7 +2112,12 @@ function update-prometheus-to-sd-parameters {
 
 # Updates parameters in yaml file for event-exporter configuration
 function update-event-exporter {
-    sed -i -e "s@{{ *event_exporter_zone *}}@${ZONE:-}@g" "$1"
+  if [[ "${MULTIZONE:-}" == "true" ]]; then
```

Comment:
It's not obvious to me this condition is actually correct.

What you're trying to achieve?
Is the purpose of that handle regional clusters? Not every multizonal one is a regional one...

Reply:
I was shadowing fix for regional clusters https://github.com/kubernetes/kubernetes/commit/81c6bb6ec2e1806d67cdde96ef49781271e9fee9 by @loburm 

We need to rethink idea of location of kubernetes cluster and introduce it not only for stackdriver.

Reply:
The test fix isn't really correct - you can take a look into:
https://github.com/kubernetes/kubernetes/pull/61892/files
to a better way of fixing that.

> We need to rethink idea of location of kubernetes cluster and introduce it not only for stackdriver.

Yes and no. In majority of cases we know exactly if we want zone or region (it's the case for pretty much all gce-related stuff). 
We need a good story for SD (and maybe couple more?, but I don't think it affects many things).

Also, there is huge difference between fixing tests and fixing production code. We can't afford breaking production code for multizonal clusters. And this PR is clearly changing that behavior.
I would like to be sure that we won't break any of those...

Reply:
I changed if so it depends on MULTIMASTER value.